### ISSUE TITLE
keep some dialogs on the screen, in a window mode (another screen)

### ISSUE THREAD

**Issue by alexeyOnGitHub** [civfanatics/CQUI_Community-Edition#228]
I suspect this one is a big ask for the current UI, but still...

say, I want to track **Great People** progress - who is available, and how many points are being generated. 
I need to remember to open a certain screen every few turns.
what if this screen (or any other popup dialog for that matter) could be made non-modal, and I can just move it to another monitor (or keep it on the same monitor on top of everything else if I choose to)?

similarly, I would really want to always keep **Civic cards** / **governments** window open on a second monitor, without the need to open it with a mouse click each time I want to check what cards I still have there. (having a hotkey for that screen would help, but that is a separate improvement).

overall, do you guys  (@Infixo , @the-m4a ) think this is even possible in the current UI architecture?

**Comment by the-m4a** [comment]
I don't think so, especially for a second monitor.  that would be pretty cool, though.

**Comment by the-m4a** [comment]
it'd be possible to leave pop-ups, I think... basically when the Great people UI is up, the main screen is "dimmed"... but that dimming is actually just a screen-sized partially transparent control that sits behind the Great People UI.  the secondary monitor, though... that's not something I think is possible?

**Comment by alexeyOnGitHub** [comment]
For something as big as great people screen, putting it on a different monitor would be more useful I think. If we can leave it on the main screen on top we should probably make it smaller and more condensed

**Comment by the-m4a** [comment]
I'm not sure what to do about this one.  For bigger screens, I can see using some of the real estate to leave up some of the things like Great People, etc.  It'd need a new UI defined, which is possible, but not yet something I've done personally... this is one of those interesting problems like the Linux was...

I played a bit in paint.net, to get a very basic idea of something that may be nice, I think?  Could probably create something that hooks into the World Tracker, like this:

![image](https://user-images.githubusercontent.com/8787640/105308810-ab66b880-5b71-11eb-9c82-aa5c5d90680a.png)

**Comment by alexeyOnGitHub** [comment]
this may be an acceptable compromise, in the context of limitations of the current UI architecture. 
although with the number of policy cards available, it would be hard to grasp what those cards mean in this view - based just on card **names**. I remember maybe ~10 card names, not more than that. adding a tooltip won't hurt, but if you need to use a tooltip to see the content of a card, I think I personally would prefer just opening the regular policies screen. 

if we were to spend time (realistically - probably your time, Michael :smiley: )  on building a solution to quickly see cards without using any additional menus and such, I would prefer creating a larger panel with more of a card info. even if we have to keep it on top and cannot move to another monitor (which would be ideal).

**Comment by Infixo** [comment]
The Civ's UI is a collection of "contexts" that kinda are placed on a screen - you can call it a window for simplicity. Within a context there is a hierarchical structure of components. It is possible to have a context visible all the time - just as any other element of the screen. I.e. the map is also a context, toolbar, minimap, etc.
Floating to another window - probably not possible, because civ windows cannot go outside the main civ window (imho). So, unless you also have a map on another screen, then probably you can't just move a window there.
